test/export/test_export.py

```diff
@@ -3463,8 +3459,6 @@ def forward(self, x):
         for op in ops:
             self.assertIn(op, (torch.ops.aten._to_copy.default,))
 
-    @testing.expectedFailureTrainingIRToRunDecomp  # T193700396
-    @testing.expectedFailureTrainingIRToRunDecompNonStrict
     def test_device_to_mutation(self):
         class Module(torch.nn.Module):
             def forward(self, x):
```

Comment:
There should be 2, 3 more instance of this task in this test.

Reply:
Oh, you mean `test_float_conversion` and `test_device_to_mutation_float`? Yeah I missed those.